src/test/java/uk/gov/hmcts/juror/api/moj/utils/JurorResponseUtilsTest.java

```diff
@@ -57,6 +66,48 @@ public void test_createMinimalPaperSummonsRecord() {
             within(10, ChronoUnit.SECONDS));
     }
 
+    @Test
+    public void updateCurrentOwnerInResponseDto() {
```

Comment:
Need test for when the juror pool is not present i.e to ensure it does not crash